services/accounts/accounts.go

```diff
@@ -31,6 +31,12 @@ type API struct {
 	feed    *event.Feed
 }
 
+type Address struct {
```

Comment:
could we avoid: `Address.Address`

Can the struct be name with something more explicit: `AddressAndPath`

Reply:
After reading the code, maybe `DerivedAddress`